cachito/web/api_v1.py

```diff
@@ -156,7 +153,6 @@ def get_requests():
     return flask.jsonify(response)
 
 
-@api_v1.route("/requests/<int:request_id>", methods=["GET"])
 def get_request(request_id):
```

Comment:
Still need to rename the parameter names to match what they are called in the OpenAPI spec, IIRC

Reply:
Ah nevermind, I guess it was only the url that needed updating

            rv["configuration_files"] = flask.url_for(
                "/api/v1.cachito_web_api_v1_get_request_config_files",
                request_id=self.id,
                _external=True,